frameworks/rooch-framework/sources/coin.move

```diff
@@ -85,6 +87,8 @@ module rooch_framework::coin {
         decimals: u8,
         /// The total value for the coin represented by coin type. Mutable.
         supply: u256,
+        /// URL for the token logo
+        icon_url: Option<string::String>
```

Comment:
Thank you. However, adding a field to a struct is a breaking change and can not be compatible with the previous version.

Maybe, we should add the `icon_url` to the dynamic field?

cc @mx819812523 @pause125 @jojoo-eth

Reply:
`icon_url` is usually used for front-end, may be we can `set_value` for CoinInfo's display object